### ISSUE TITLE
Schedule: Find out project ideas and how we would like to tackle this summer

### ISSUE THREAD

**Issue by map34** [map34/sda-projects-hub#4]
# Objective

Let's find out how we want to utilize our summer for projects

* Should we have a weekly meeting? Discuss about project ideas, what's confusing, career prospects, pretty much anything. Note that this can be super casual, like just get coffee somewhere and just talk about stuff, or work together on projects.
* Bi-weekly/monthly projects maybe on different technologies?
* Can we integrate hardware/gadgets into our projects? -> Useful for understanding the backgrounds on how software systems interact with each other
* Projects that you really want to own and be responsible for?

# Reason

It is really easy to forget about what we learn in school, and sometimes school doesn't teach everything you would like to learn. This group is not to replace classes (obviously), but to act as a complementary supplement to classes. What I found out during school is that I wish I learned some of these tech stacks while being in school, because the industry (if you are going that way) expects you to have some of these technologies under your belt. Have you ever heard "Requires 20 years of React experience?" Like what? React has been only out for 4 years.

Also, let's not forget about the fundamentals. We gotta keep sharp on algorithms and data structures because those are core fundamentals, without them, we fall to ashes. Also, let's try to understand both low-level and high-level software systems. One wise professor in Stanford (yup, Marty Stepp) told me that most of engineers nowadays don't care about how hardware works because they only do software. But, memory leaks? virtual/physical memory? L1/L2 caches? those all are important concepts that make software to not only work but work fast.

**Comment by dwinasolihin** [comment]
Just reading memory leaks, virtual/physical memory, L1/L2 cache brought me back to my OS and hardware days and I just cry a little on the inside...

**Comment by scaraclette** [comment]
I like the idea of weekly code and chill! + I'm planning to start a portfolio of code projects for future internships and maybe jobs.

**Comment by map34** [comment]
Yeah, btw, this is not limited to only learning, we can talk about job huntings, networking events to get jobs, etc.

**Comment by dwinasolihin** [comment]
Yes, find me a job! 💃

**Comment by map34** [comment]
LOL, main reason to have this too

**Comment by dwinasolihin** [comment]
Who is still interested on the idea of a smart mirror? 100% Hotter definitely inspired me about that

**Comment by scaraclette** [comment]
I'll skip the raspberry pi, I'm planning on doing python again and amp up java skills to make phone apps

**Comment by map34** [comment]
haha 100% hotter, I mean why not

**Comment by map34** [comment]
@scaraclette if you want to build an Android app with Java, use [Android Studio](https://developer.android.com/studio/) it's the best environment for Android thus far. A new language called Kotlin has also emerged so you might want to check that out for Android. For iOS, just Swift :P.

**Comment by map34** [comment]
@dwinasolihin @scaraclette ok I know Scarlett has finals next week, when are you guys ready to start/discuss?

**Comment by lsetiawan** [comment]
I am looking for frontend and testers/code reviewers :wink: If anyone is available, let's chat and I will pass along your resume!

**Comment by lsetiawan** [comment]
@map34 I got wedding stuff then honeymoon. Won't have time to do any projects stuff until July 15th! :smile:
